src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapperTest.java

```diff
@@ -52,53 +53,77 @@ void setUp() {
         lenient().when(mockRequester.getId()).thenReturn(SOME_UUID);
         pathologyRecordBuilder = PathologyRecord.builder()
-            .requester(mockRequester)
+            .requestingPractitioner(mockRequester)
             .patient(mock(Patient.class));
     }
 
     @Test
-    void testMapPathologyRecordToBundleWithPractitioner() {
+    void testMapPathologyRecordToBundleWithPatient() {
+        final var mockPatient = mock(Patient.class);
+        pathologyRecordBuilder.patient(mockPatient);
+
+        final Bundle bundle = bundleMapper.mapToBundle(pathologyRecordBuilder.build());
+
+        final var patientBundleEntries = bundle.getEntry().stream()
+            .filter(entry -> entry.getResource() instanceof Patient)
+            .collect(Collectors.toList());
+        final var patients = patientBundleEntries.stream()
+            .map(BundleEntryComponent::getResource)
+            .map(Patient.class::cast)
+            .collect(Collectors.toList());
+
+        assertAll(
+            () -> verifyBundle(bundle),
+            () -> assertThat(patients).hasSize(1).contains(mockPatient),
+            () -> assertThat(patientBundleEntries).first()
+                .extracting(BundleEntryComponent::getFullUrl)
+                .isEqualTo(FULL_URL)
+        );
+    }
+
+    @Test
+    void testMapPathologyRecordToBundleWithRequester() {
         final var mockRequester = mock(Practitioner.class);
-        pathologyRecordBuilder.requester(mockRequester);
+        pathologyRecordBuilder.requestingPractitioner(mockRequester);
 
         final var bundle = bundleMapper.mapToBundle(pathologyRecordBuilder.build());
 
-        final var practitionerBundleEntries = bundle.getEntry().stream()
+        final var requesterBundleEntries = bundle.getEntry().stream()
             .filter(entry -> entry.getResource() instanceof Practitioner)
             .collect(Collectors.toList());
-        final var practitioners = practitionerBundleEntries.stream()
+        final var practitioners = requesterBundleEntries.stream()
             .map(BundleEntryComponent::getResource)
             .map(Practitioner.class::cast)
             .collect(Collectors.toList());
 
         assertAll(
             () -> verifyBundle(bundle),
             () -> assertThat(practitioners).hasSize(1).contains(mockRequester),
-            () -> assertThat(practitionerBundleEntries).first()
+            () -> assertThat(requesterBundleEntries).first()
                 .extracting(BundleEntryComponent::getFullUrl)
                 .isEqualTo(FULL_URL)
         );
     }
 
     @Test
-    void testMapPathologyRecordToBundleWithPatient() {
-        final var mockPatient = mock(Patient.class);
-        pathologyRecordBuilder.patient(mockPatient);
+    void testMapPathologyRecordToBundleWithRequestingOrganization() {
+        final var mockRequestingOrganization = mock(Organization.class);
+        pathologyRecordBuilder.requestingOrganization(mockRequestingOrganization);
```

Comment:
Could we change this to use the `.performingOrganization` field from the builder?

```suggestion
        pathologyRecordBuilder.performingOrganization(mockRequestingOrganization);
```

Reply:
Apologies - this comment is incorrect

Reply:
This is testing a pathology record (with a requesting organization) maps properly to a bundle. The `performingOrganization` method shouldn't be used here then

Reply:
Yeah, I made the comment then realised 😦